packages/core/lib/blockUtils.js

```diff
@@ -1,5 +1,6 @@
 const _ = require('lodash')
 const webpackMerge = require('webpack-merge')
+const webpackBlocksMerge = webpackMerge.smartStrategy({ 'resolve.extensions': 'prepend' })
```

Comment:
I would say we should rather put this into [resolve()](https://github.com/andywer/webpack-blocks/blob/release/1.0/packages/webpack/index.js).